x-pack/test/api_integration/config.js

```diff
@@ -26,9 +26,6 @@ export async function getApiIntegrationConfig({ readConfigFile }) {
         ...xPackFunctionalTestsConfig.get('kbnTestServer.serverArgs'),
         '--xpack.security.session.idleTimeout=3600000', // 1 hour
         '--optimize.enabled=false',
-        '--xpack.endpoint.enabled=true',
```

Comment:
I don't believe we need these as they are pulled in from the functional `config.js`